### DISCUSSION TITLE
JSON_EXTRACT example not working on DBeaver mysql version 8.0.33

### DISCUSSION THREAD

**Discussion by MarcusR1** [dbeaver/dbeaver#20224] [Q&A]
### Description

I have tried executing a simple JSON command in DBeaver but it doesn't work. I have veriefed that it works in phpMyAdmin cPanel MySQL editor and it works. I have also reached out to the StackOverflow community who have helped me verify that it looks like a bug in DBeaver, plesae see https://stackoverflow.com/questions/76380997/json-extract-example-not-working-on-dbeaver-mysql-version-8-0-33

### DBeaver Version

Community 8.0.33

### Operating System

Ubuntu on Digital Ocean

### Database and driver

com.mysql.cj.jdbc.Driver
Driver for MySQL 8 and later

mysql-connector-java  8.0.29

### Steps to reproduce

I'm trying to get JSON_EXTRACT to work on a Digital Ocean droplet, executing

select version();

gives me 8.0.33-0ubuntu0.20.04.2

And I'm trying to use the example from the documentation that says it should work https://dev.mysql.com/doc/refman/8.0/en/json-search-functions.html#function_json-contains

but the code I pasted from the example:

SET @j = '{"a": 1, "b": 2, "c": {"d": 4}}';
SET @j2 = '1';
SELECT JSON_CONTAINS(@j, @j2, '$.a');

returns NULL

**Comment by LonwoLonwo** [comment]
Hello @MarcusR1 

Please, provide clear steps to reproduce your example.
For now, I can not reproduce it, unfortunately.

![2023-06-05 17_49_40-Window](https://github.com/dbeaver/dbeaver/assets/45152336/dfc4c18c-1484-4d61-b4f8-bee1e8c63caa)

**Comment by MarcusR1** [comment]
Hi Anastasiya,

Did you read the StackOverflow post? There are screen shots from my DBeaver, and RiggsFolly (https://stackoverflow.com/users/2310830/riggsfolly) also tried it and it failed for him too. If you need logs from DBeaver, please send me instructions as to how to get them.

Best regards, Marcus

**Comment by E1izabeth** [comment]
Unfortunately, I can't reproduce it too.
Maybe logs will help us. Could you send them? The instructions are here https://dbeaver.com/docs/wiki/Log-files/

**Comment by LonwoLonwo** [comment]
And please provide a video of your issue reproducing. Then I will see your steps.

**Comment by MarcusR1** [comment]
![dbeaver_sboPkDgNuj](https://github.com/dbeaver/dbeaver/assets/77488019/6306543b-a2bb-46cf-98d2-bcda3f7eac84)

Here's a video, the logs don't show anything.

**Comment by LonwoLonwo** [comment]
But you didn't execute "set" statements first. The execute button only executes a current query.
You need to execute "set" statements before your select.

**Comment by E1izabeth** [comment]
Please let me know if executing the following set of statements as a script with `Execute Script` button or sequantialy executing them one-by-one with `Execute SQL query` button helps.

**Comment by MarcusR1** [comment]
Execute Script gives the correct response of 1
My apologies, I was not executing the whole script as one.

**Comment by E1izabeth** [comment]
Thank you for update
